.github/workflows/build-binaries.yml

```diff
@@ -78,7 +76,7 @@ jobs:
         run: |
           docker buildx build -f arm-compile.Dockerfile --platform ${{ matrix.platform }} \
           --build-arg ARCH=${{ matrix.arch }} \
-          --build-arg FETCH_CMD="mkdir -p ~/.pkg-cache/v2.5 && cd ~/.pkg-cache/v2.5 && wget https://github.com/robertsLando/pkg-binaries/releases/download/v1.0.0/fetched-v10.15.3-linux-${{ matrix.arch }} -O fetched-v10.4.1-linux-${{ matrix.arch }}" \
+          --build-arg FETCH_CMD="mkdir -p ~/.pkg-cache/v2.5 && cd ~/.pkg-cache/v2.5 && wget https://github.com/robertsLando/pkg-binaries/releases/download/v1.0.0/fetched-12.18.1-linux-${{ matrix.arch }} -O fetched-v12.18.1-linux-${{ matrix.arch }}" \
```

Comment:
There isn't a `12.18.1` binary on that page. It looks like you should get `fetched-v12.2.0-linux-armv7`.